data/00400_ldu.js

```diff
@@ -7,7 +7,13 @@ exports.seed = function (knex, Promise) {
     .then(function (regionId) {
       // Inserts seed entries
       return knex(tableName).insert([
-        { description: 'test ldu', region_id: regionId.id }
+        { description: 'test ldu 1', region_id: regionId.id },
+        { description: 'test ldu 2', region_id: regionId.id },
+        { description: 'test ldu 3', region_id: regionId.id },
+        { description: 'test ldu 4', region_id: regionId.id },
+        { description: 'test ldu 5', region_id: regionId.id },
+        { description: 'test ldu 6', region_id: regionId.id },
+        { description: 'test ldu 7', region_id: regionId.id }
```

Comment:
Only one LDU will have teams etc, so on the region graph there will only be one line? Can we add more teams to one other LDU?